levelkdev/swapr-dapp
# update xdai logo

# Summary

Closes #685 

To test:
1. Connect to Gnosis Chain
2. Go to Rewards and find campaigns with `XDAI`
3. Make sure that new logo appears
